### ISSUE TITLE
ERROR: Can not combine '--user' and '--target'

### ISSUE THREAD

**Issue by SgtFlex** [Sigmmma/mek#15]
Been trying to get MEK tools to work. I keep running into errors, and I believe both Python and Pip is installed. I don't know if I've done something wrong on my end, but any help would be greatly appreciated.

https://pastebin.com/Ssy57YX5

**Comment by Mimickal** [comment]
Can you go step by step on what you did to install this?

**Comment by SgtFlex** [comment]
> Can you go step by step on what you did to install this?

Yes, I made a quick 2 minute video showing exactly what I did.

https://www.youtube.com/watch?v=r0ucsoZtxCM

**Comment by ShellyHerself** [comment]
Windows is doing something weird. Could you untick the box called `portable install` and try installing again?

**Comment by SgtFlex** [comment]
> Windows is doing something weird. Could you untick the box called `portable install` and try installing again?

I tried with "portable install" unticked, something about a permissions issue I think? Is the default install location suppossed to be the System32 folder?

Anyways, here's the full log (got too big for pastebin) 

https://drive.google.com/open?id=1I2eq_o202Hv3umxY_ZY3gCqwaElh7pYe

**Comment by SgtFlex** [comment]
So I tried running MEK_Installer.pyw through an Admin CMD window, and things went smoother but I think some things still aren't working. I can't open Mozzarilla normally as it just won't open unless I explicitly open it with an Admin CMD window. I mainly need the animation_decompressor in Mozzarilla, and when I tried to use it, it just gave me this error log.

https://pastebin.com/q7jLntgQ

Also when closing pool I get this error as well: https://i.imgur.com/ZRSyDTw.png

I think something weird is going on and things are possibly not installing correctly for some reason?

**Comment by ShellyHerself** [comment]
The Pool bug we'll look into later. The first bug is because spv3 removes animation data from some animations.  It can't decompress them because the spv3 tags are simply invalid.

**Comment by SgtFlex** [comment]
> The Pool bug we'll look into later. The first bug is because spv3 removes animation data from some animations. It can't decompress them because the spv3 tags are simply invalid.

Hrm, that's unfortunate. I know this might not be the place to ask anymore since it sounds like my issue might be resolved then (aside from oddly not being able to start Mozzarilla without using an Admin CMD), but is there anyway around that or some other tool for dealing with this?

**Comment by ShellyHerself** [comment]
We're actually updating stuff right now to fix this issue

**Comment by SgtFlex** [comment]
> We're actually updating stuff right now to fix this issue

That's fantastic! Thanks for the assistance, and hopefully that'll work for what I need when it gets updated.

**Comment by ShellyHerself** [comment]
We've updated it. Run the installer again (without "portable install") and make sure the install path isn't system32.

You should under no circumstance install in system32.

**Comment by SgtFlex** [comment]
So Mozzarilla seems to run now without me needing to use CMD on it, but the animation decompressor isn't working for me unfortunately. I wasn't sure if the update was to just fix the issues I was having or an update to the animation decompressor as well.

I've included a short gif just to demonstrate how I tried to use it alongside the error I received. 

https://gfycat.com/IncredibleIckyBullmastiff

https://pastebin.com/GSyU1dby

Should I create a new issue for this or via email or just here? I apologize if there's some sort of readme or I am misunderstanding the process, I'm still relatively new to this in some ways.

**Comment by ShellyHerself** [comment]
The animation decompressor can't work with those spv3 animations because they have nulled out a lot of the animations. The ones that are valid should be in the decompressed version. We'll add a check for this specific thing later.

You can join https://discord.reclaimers.net for more direct help.

**Comment by SgtFlex** [comment]
Bummer, but thanks for all the assistance!
